src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/EmailAddressAdder.tsx

```diff
@@ -134,7 +134,6 @@ const EmailAddressAddForm = () => {
             setHasPressedButton(true);
           }}
           isLoading={hasPressedButton}
-          aria-live="polite"
```

Comment:
I copied this (and the changes to `<Button>`) from https://github.com/mozilla/blurts-server/pull/4203 because it was also relevant to this PR.